format-files.py

```diff
@@ -0,0 +1,43 @@
+#   A wrapper meant for automating the format process, using clang-format.
```

Comment:
I'd request that you not have this tool here as its job can literally be done with a one-liner:

```
find src -type f -name "*.[ch]*" | xargs -n1 clang-format -i
```

Reply:
feel free to remove it then